context/app/static/js/components/error/ErrorBody/ErrorBody.spec.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import { render, screen } from 'test-utils/functions';
+import ErrorBody from './ErrorBody';
+
+const help = { url: 'mailto:help@hubmapconsortium.org', name: 'help@hubmapconsortium.org' };
+const login = { url: '/login', name: 'login' };
+
+test('maintenance page', () => {
+  render(<ErrorBody isMaintenancePage />);
+  expect(screen.getByText('While the portal is under maintenance', { exact: false })).toBeInTheDocument();
+  expect(screen.getByRole('link', { name: 'HuBMAP Consortium' })).toHaveAttribute(
+    'href',
+    'https://hubmapconsortium.org/',
+  );
+});
+
+test('Globus 401', () => {
+  render(<ErrorBody errorCode={401} isGlobus401 />);
+  expect(screen.getByText('Your credentials have expired.', { exact: false })).toBeInTheDocument();
+  expect(screen.getByRole('link', { name: login.name })).toHaveAttribute('href', login.url);
+});
+
+test('401', () => {
+  render(<ErrorBody errorCode={401} />);
+  expect(
+    screen.getByText('You have not been added to the HuBMAP Group on Globus.', { exact: false }),
+  ).toBeInTheDocument();
+  expect(screen.getByRole('link', { name: help.name })).toHaveAttribute('href', help.url);
+});
+
+test('403', () => {
+  render(<ErrorBody errorCode={403} />);
+  expect(screen.getByText('You may not have access to this resource', { exact: false })).toBeInTheDocument();
+  expect(screen.getByRole('link', { name: login.name })).toHaveAttribute('href', login.url);
+});
+
+test('404', () => {
+  render(<ErrorBody errorCode={404} />);
+  expect(screen.getByText('If this page should exist,', { exact: false })).toBeInTheDocument();
+  expect(screen.getByRole('link', { name: help.name })).toHaveAttribute('href', help.url);
+});
+
+test('all other statuses', () => {
+  render(<ErrorBody errorCode={500} />);
+  expect(screen.getByText('If this problem persists,', { exact: false })).toBeInTheDocument();
+  expect(screen.getByRole('link', { name: help.name })).toHaveAttribute('href', help.url);
+});
```

Comment:
I'm not sure that I'll keep up with you, but I really respect the level of detail you put into the tests. Thanks!